resources/sdk/purecloudjava/templates/pom.mustache

```diff
@@ -223,11 +223,21 @@
     <artifactId>jaxb-api</artifactId>
     <version>2.2.12</version>
 </dependency>
- <!-- Logging: SLF4J -->
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-      <version>${slf4j-version}</version>
+      <groupId>org.ini4j</groupId>
+      <artifactId>ini4j</artifactId>
+      <version>0.5.4</version>
```

Comment:
You can put the versions for the added dependencies down below in the section with all the other dependency versions. So this one would become ${ini4j-version} for example.